src/Sylius/Bundle/ApiBundle/Exception/PaymentMethodNotFoundException.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Exception;
+
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+/** @experimental */
+final class PaymentMethodNotFoundException extends NotFoundHttpException
```

Comment:
It should not be the `NotFoundHttpException` as it might be used in a non-HTTP context. We usually use the `\RuntimeException`.